README.md

```diff
@@ -3,17 +3,28 @@
 
 [![Build Status](https://circleci.com/gh/iodide-project/pyodide.png)](https://circleci.com/gh/iodide-project/pyodide)
 
-The Python scientific stack, compiled to WebAssembly.
+Pyodide is cPython compiled to WebAssembly and running in the browser.  It can run any pure Python package, and 
+includes over 37 of Python's [best Data Science libraries](./packages) compiled to WebAssembly.
 
-It provides transparent conversion of objects between Javascript and Python.
-When inside a browser, this means Python has full access to the Web APIs.
+Pyodide works great with Javascript. Proxy objects make it easy for Python and Javascript to interoperate. 
+Pyodide provides transparent conversion of objects between Javascript and Python.
+When inside a browser, this means Python has full access to the Window and Document objects.
```

Comment:
The `window` and `document` objects are just a small part of the Web API that Pyodide has access to.  Why limit it to just those two?  If the concern that the phrase "Web API" isn't well-known, maybe link to: https://developer.mozilla.org/en-US/docs/Web/API